src/components/header/buttons/PeersButton.js

```diff
@@ -89,13 +109,16 @@ export default class PeersButton extends Component {
 }
 
 PeersButton.propTypes = {
-  peerGroup: PropTypes.object
+  peerGroup: PropTypes.object,
+  alias: PropTypes.string,
+  onAliasChange: PropTypes.func
```

Comment:
Can remove `onAliasChange`, looks like it's only ever a local method rather than an expected prop.

Reply:
It's being used to communicate definite alias change to the parent, passed in here https://github.com/ipfs-shipyard/peerpad/pull/116/files#diff-96eeb67cd2b0e6e2fe9df14e4c8036f3R153